x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/processor_form/processors/attachment.tsx

```diff
@@ -0,0 +1,202 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiCode, EuiFlexGroup, EuiFlexItem, EuiSpacer } from '@elastic/eui';
+
+import {
+  ComboBoxField,
+  FIELD_TYPES,
+  UseField,
+  ToggleField,
+  Field,
+} from '../../../../../../shared_imports';
+
+import { FieldNameField } from './common_fields/field_name_field';
+import { TargetField } from './common_fields/target_field';
+import { IgnoreMissingField } from './common_fields/ignore_missing_field';
+import { FieldsConfig, to, from } from './shared';
+
+const propertyValues: string[] = [
+  'content',
+  'title',
+  'author',
```

Comment:
I think this should also contain the `name` property.

Reply:
I initially thought the same, but name is not allowed by ES. The actual supported properties are listed a little bit down below https://www.elastic.co/guide/en/elasticsearch/reference/master/attachment.html#attachment-fields

Reply:
Oh I see...probably this is a mistake in the docs then.